inspire/modules/formatter/views.py

```diff
@@ -47,3 +47,28 @@ def get_latex(recid, latex_format):
     record = get_record(recid)
     latex = Latex(record, latex_format).format()
     return jsonify({"result": latex, "recid": recid})
+
+
+@blueprint.route("/download-bibtex/<int:recid>")
+def get_bibtex_file(recid):
+    record = get_record(recid)
+    results = Bibtex(record).format()
+    generator = (cell for row in results
```

Comment:
What is the format returned by `Bibtex(record).format()` ? It does not return the text ready to be returned?